test/parallel_api/ranges/range.zip/zip_view.ctor.default.pass.cpp

```diff
@@ -0,0 +1,93 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+// zip_view() = default;
+
+#include "support/test_config.h"
+#include "support/utils.h"
+
+#if _ENABLE_STD_RANGES_TESTING
+
+#include <ranges>
+
+#include <cassert>
+#include <type_traits>
+#include <utility>
+
+#include <oneapi/dpl/ranges>
+
+namespace dpl_ranges = oneapi::dpl::ranges;
+
+#if 1
+template <typename... Types>
+using tuple_type = oneapi::dpl::__internal::tuple<Types...>;
+#else
+using tuple_type = std::tuple<Types...>;
+#endif
```

Comment:
Did you mean here that std::tuple branch should be enabled in the future, or is it just dead code?